core/src/main/java/com/netflix/conductor/core/reconciliation/WorkflowSweeper.java

```diff
@@ -78,20 +83,27 @@ public void sweep(String workflowId) {
             WorkflowContext workflowContext = new WorkflowContext(properties.getAppId());
             WorkflowContext.set(workflowContext);
             LOGGER.debug("Running sweeper for workflow {}", workflowId);
-
+            if (!executionLockService.acquireLock(workflow.getWorkflowId())) {
```

Comment:
here it will always be npe, should be just workflowId

Reply:
Resolved thanks for the catch !